src/rendering/mod.rs

```diff
@@ -10,6 +10,13 @@ pub struct Vertex {
     pub color: [f32; 4],
     pub size: f32,
 }
+
+pub struct Line {
+    pub indicies: Vec<u32>,
```

Comment:
You mentioned potentially moving this out of the struct. Can you make an issue for that?

Reply:
done https://github.com/dr-sg/rscat/issues/13